packages/stream/index.js

```diff
@@ -34,7 +34,7 @@ const freeze = /** @type {<T>(v: T | Readonly<T>) => T} */ (Object.freeze);
  */
 export const makeQueue = () => {
   let { promise: tailPromise, resolve: tailResolve } = makePromiseKit();
-  return {
+  return Far('AsyncQueue', {
```

Comment:
Kits are not useful when they are declared as `Far`, since that means the kit is a single ocap, and when sent to a different vat, it has no locally-separable methods.

I'm picking on `makeQueue` here, but it applies to the other changes in this PR.

Presences can be used like `E(queueP).get()` and `E(queueP).put(val)`, but `const { get, put } = await queueP;` just makes `get` and `put` `undefined`.

However, if you instead make it a hardened record of far functions (which doesn't change the local behaviour of your code), like:
```js
return harden({
   get: Far('AsyncQueueGet', () => { ... }),
   put: Far('AsyncQueuePut', (value) => { ... }),
});
```

then that's what I meant earlier by leaning into far functions.  The remote usage would be:
```
const { get: getP, put: putP } = await queueP;
await E(getP)();
await E(putP)(value);
```

Again, some TODOs would be sufficient, since without a captp test of these abstractions we can't verify their remote behaviour is correct.